README.rst

```diff
@@ -122,8 +128,8 @@ On MacOS, we recommend using the latest version of ``clang++`` and ``libomp``:
 
     $ brew install llvm libomp
 
-The Lightning-GPU backend has several dependencies (e.g. ``CUDA``, ``custatevec-cu12``, etc.), and hence we recommend referring to Lightning-GPU installation section.
-Similarly, for Lightning-Kokkos it is recommended to configure and install Kokkos independently as prescribed in the Lightning-Kokkos installation section.
+The Lightning-GPU backend has several dependencies (e.g. ``CUDA``, ``custatevec-cu12``, etc.), and hence we recommend referring to `Lightning-GPU installation`_ section.
```

Comment:
Does not seem to [render](https://xanaduai-pennylane--851.com.readthedocs.build/projects/lightning/en/851/lightning_qubit/installation.html) correctly.

Reply:
:open_mouth:  I did not know that the README was linked to the website. I will check if it is possible to find a good solution for this hyperlink